sound/hybrid_font.h

```diff
@@ -799,8 +802,58 @@ class HybridFont : public SaberBase, public Looper {
 	SaberBase::DoEffect(EFFECT_POSTOFF, saved_location_);
       }
     }
+    // Wait for retraction sounds to finish, then play idle.wav
+#ifdef ENABLE_IDLE_SOUND
+    if (idling_) {
+      if (GetWavPlayerPlaying(&SFX_in) || GetWavPlayerPlaying(&SFX_pstoff)) {
+        return;
+      } else {
+        idling_ = false;
+        StartIdleSound();
+      }
+    }
+#endif
   }
-  bool swinging_ = false;
+  void CheckStopIdle() {
+#ifdef ENABLE_IDLE_SOUND
+    // Since idle sound re-uses hum_player,this will prevent
+    // a swing command from triggering monophonic hum while SaberBase is OFF.
+    if (!SaberBase::IsOn() && !GetWavPlayerPlaying(&SFX_idle)) {
```

Comment:
Since it's always going to be the hum_player playing SFX_idle, we can do hum_player_ && hum_player->isPlaying() && hum_player->current_file_id().GetEffect() == &SFX_idle instead.  (longer, but also much faster.)

Reply:
This works great, and I see it's more efficient than running through all the wavplayers hunting for idle. 
But, it seems it's not even needed. See above.